tests/integration/corrupt-dump-fuzzer.tcl

```diff
@@ -121,7 +121,7 @@ foreach sanitize_dump {no yes} {
                 if { [catch { r restore "_$k" 0 $dump REPLACE } err] } {
                     set restore_failed true
                     # skip if return failed with an error response.
-                    if {[string match "ERR*" $err]} {
+                    if {[string match "ERR *" $err]} {
```

Comment:
p.s. many of these changes, like this one, are really unimportant.
we're not trying to match some specific error, just distinguish between an error response, and a disconnection due to a crash.

Reply:
I'll concede these probably aren't useful, I'll remove all of the "ERR*" checks.